frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl

```diff
@@ -38,9 +38,15 @@
 
 <#assign preferences = freeMarkerPortletPreferences.getPreferences({"portletSetupPortletDecoratorId": "sw360", "destination": "/search"}) />
 <#assign login_css = is_signed_in?then("signed-in", "not-signed-in") />
+<#assign liferay_user = themeDisplay.getUser() />
+<#assign js_folder = theme_display.getPathThemeJavaScript() />
+<#assign js_banner_file = htmlUtil.escape(portalUtil.getStaticResourceURL(request, "${js_folder}/banner.js")) />
+
 <html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
 
 <head>
+	<script type="text/javascript" src ="${js_banner_file}" > </script>
+
```

Comment:
Please remove this script tag, as banner JS has been removed. also at line `42` and `43`.